editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1398,7 +1419,9 @@ bool CanvasItemEditor::_gui_input_rotate(const Ref<InputEvent> &p_event) {
 					drag_type = DRAG_ROTATE;
 					drag_from = transform.affine_inverse().xform(b->get_position());
 					CanvasItem *canvas_item = drag_selection[0];
-					if (canvas_item->_edit_use_pivot()) {
+					if (temp_pivot != Vector2(INFINITY, INFINITY)) {
```

Comment:
I'm not sure you can rely on this to work in a floating point comparison. That's why we have to use things like `isinf()` instead of equality operators for scalars, so that's probably the same for vectors.